spec/orders.mediawiki

```diff
@@ -815,6 +815,21 @@ The client will respond with an acknowledgement.
 The taker will get the key from the maker's redemption and broadcast their own
 redemption transaction.
 
+It is also possible for an  order to go through the matching cycle without
+generating a match. This will be common for limit orders, but can also occur for
```

Comment:
<s>Common for limit orders with time-in-force "immediate" would be clearer I think.</s>  Oh, were you thinking `nomatch` would be sent for limit orders that get booked too?  That seems a little unnecessary because the client does not need to take any action for unmatched but booked limit orders.

Reply:
It actually solves a small problem that I've been cheating around. [See `(*trackedTrade).processEpoch`](https://github.com/decred/dcrdex/blob/f3e94af5e89df7342c8cf5c70ef7877259284f53/client/core/trade.go#L1143-L1160). For standing limit orders, I'm setting them as booked as soon as I know their epoch has passed, but that isn't necessarily accurate, since if they fill they are never actually booked. Having a `nomatch` notification allows me to set them as booked confidently in either `negotiate` or via the upcoming `handleNoMatch` function.